src/m365/aad/commands/m365group/m365group-remove.ts

```diff
@@ -115,6 +109,124 @@ class AadM365GroupRemoveCommand extends GraphCommand {
       }
     }
   }
+
+  private async getM365GroupSiteURL(logger: Logger, id: string): Promise<string> {
+    if (this.verbose) {
+      await logger.logToStderr(`Getting the site URL of Microsoft 365 Group: ${id}...`);
+    }
+
+    const requestOptions: CliRequestOptions = {
+      url: `${this.resource}/v1.0/groups/${id}/drive?$select=webUrl`,
+      headers: {
+        accept: 'application/json;odata.metadata=none'
+      },
+      responseType: 'json'
+    };
+
+    const res = await request.get<{ webUrl: string }>(requestOptions);
+    return res.webUrl.substring(0, res.webUrl.lastIndexOf('/'));
+  }
+
+  private async deleteM365GroupSite(logger: Logger, url: string): Promise<void> {
+    if (this.verbose) {
+      await logger.logToStderr(`Deleting the group site: '${url}'...`);
+    }
+
+    const spoAdminUrl = await spo.getSpoAdminUrl(logger, this.debug);
+    this.spoAdminUrl = spoAdminUrl;
```

Comment:
Since we need this URL in multiple methods, I'd rather not retrieve it inside this method, but take it a level up. Otherwise, if ever changed the execution order, it could lead to issues, so let's get ahead of it. Also, let's avoid the unnecessary assignment to a local const and then to a class property, and assign to class property directly

Reply:
Moved the function call a level up, removed the class property, and passed it in functions to keep it consistent with siteUrl.